Dockerfile

```diff
@@ -12,7 +12,7 @@
 ################################
 
 # Debian based base image for Node.js
-FROM node:18.16.0 
+FROM node:18.16.0-bullseye
```

Comment:
Small request -- can you add a comment here about why we switched to the bullseye version and when/if we should consider updating

Reply:
Yes, that would be a good place for it!